src/main/java/seedu/address/storage/ReminderStorage.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.storage;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import seedu.address.model.event.Description;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.Reminder;
+import seedu.address.model.event.Time;
+
+public class ReminderStorage {
```

Comment:
This class should be part of StorageManager as per addressbook, calendar, userPrefs, tagtree. Initial read should be part of MainApp init() method. 
![image](https://user-images.githubusercontent.com/69659433/97466373-fe1f4280-197d-11eb-98d3-fbed92260a36.png)